content/en/docs/FAQ/authentication.md

```diff
@@ -16,7 +20,13 @@ For example, if you want to log into Kiali using Kiali's own service account, yo
 kubectl get secret -n istio-system $(kubectl get sa kiali-service-account -n istio-system -o "jsonpath={.secrets[0].name}") -o jsonpath={.data.token} | base64 -d
 ```
 
-Note that this example assumes you installed Kiali in the `istio-system` namespace.
+**For kubernetes v1.24+**
+
+You can request a short lived token for a service account by issuing the following command:
+
+> kubectl -n istio-system create token kiali-service-account
```

Comment:
for markdown, you need to use triple-backquotes to format this command properly. So this should be this (ignore the backslashes - I can't figure out how to embed the triple-backquotes inside a suggestion - it should be triple backquotes, then newline, then the kubectl command, then newline, then another triple backquote):

```suggestion
\```
kubectl -n istio-system create token kiali-service-account
\```
```